webhook_to_fedora_messaging/models/util.py

```diff
@@ -37,4 +24,4 @@ class CreatableMixin:
     An SQLAlchemy mixin to store the time when an entity was created
     """
 
-    creation_date = Column("creation_date", SQLDateTime, nullable=False, server_default=utcnow())
+    creation_date = Column("creation_date", SQLDateTime, nullable=False, server_default=str(partial(datetime.now, tz=UTC)))
```

Comment:
I don't think that this is going to work, we can't rely on Python's conversion of dates to string to be a valid SQL statement. 
We could however use `default=partial(datetime.now, tz=UTC)` instead of `server_default`, since it expects a python callable.